### PR TITLE
(Re-resubmission) Add shared-clipboard workshop

### PR DESCRIPTION
**Pull request by quackduck**
Made more requested changes!

### CODE REVIEW

File: workshops/shared_clipboard/README.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ pkg install termux-api
 
 ## Building it!
 
-Open up your favorite editor, make a new file titled `myclip.go` (you can name it whatever you want, really) and add in
+Open up your favorite editor (I recommend VS Code and GoLand), make a new file titled `myclip.go` (you can name it whatever you want, really) and add in
```
Comment (sampoder) [inline workshops/shared_clipboard/README.md:73]: Maybe use or?

**Review by sampoder — APPROVED** [review]
:tada: 👾 😄 YAY! Congrats!

File: workshops/shared_clipboard/README.md (hunk 1 above)
Comment (quackduck) [reply workshops/shared_clipboard/README.md:73]: But I recommend both lol. Here "and" is more appropriate.

Comment (quackduck) [reply workshops/shared_clipboard/README.md:73]: If I was saying "I recommend using..." then "or" would be better

Comment (sampoder) [reply workshops/shared_clipboard/README.md:73]: k

**Comment by quackduck** [comment]
Who is this god you speak of?

**Comment by sampoder** [comment]
@MatthewStanciu

**Comment by quackduck** [comment]
Would you like me to squash all the commits?

**Comment by sampoder** [comment]
Nah, it's all good. We squash on merge :)

**Comment by quackduck** [comment]
So I just need to wait now, correct? Can I do anything else to make it better?

**Comment by sampoder** [comment]
Nothing you need to do, Matthews doing his best to go over all the workshops this week. If you see a mistake or an improvement area though, feel free to improve it but you don't need to

**Comment by MatthewStanciu** [comment]
Wow, great work. FYI I made a few changes:

* I removed the first GIF at the beginning because it was broken (giphy links always break for some reason). You're welcome to add it back via PR
* I removed the GIF of Trump because I don't like him
* I noticed a few tiny formatting errors so I went ahead and fixed those
* I also added a disclaimer that this is a long workshop and not meant for beginners

This is a suuuuper heavy workshop and I'm glad most workshops aren't this long, but I think this is an incredible addition to the workshops site. Great job :)

**Comment by quackduck** [comment]
Thanks for that! I don't like Trump either, just saying. 😄

**Pull request merged**